crates/cli_bin/tests/apply.rs

```diff
@@ -2852,6 +2852,68 @@ fn apply_stdin_simple() -> Result<()> {
     Ok(())
 }
 
+/// simple stdin example from documentation but with js array append query/operation
+#[test]
+fn apply_stdin_js_array_append() -> Result<()> {
+    let (_temp_dir, fixture_dir) = get_fixture("limit_files", false)?;
+
+    let input_file = r#"const allLanguages = ["python", "java",]"#;
+    let expected_output = r#"const allLanguages = ["python", "java", "kotlin",]"#;
+
+    let mut cmd = get_test_cmd()?;
+    cmd.arg("apply")
+        .arg(r#"`const allLanguages = [$langs]` where { $langs += `"kotlin",` }"#)
+        .arg("--stdin")
+        .arg("--lang")
+        .arg("js")
+        .current_dir(&fixture_dir);
+
+    cmd.write_stdin(String::from_utf8(input_file.into())?);
+
+    let result = cmd.output()?;
+
+    let stderr = String::from_utf8(result.stderr)?;
+    println!("stderr: {:?}", stderr);
+    let stdout = String::from_utf8(result.stdout)?;
+    println!("stdout: {:?}", stdout);
+
+    assert!(result.status.success(), "Command should have succeeded");
+    assert!(stdout.contains(expected_output));
+
+    Ok(())
+}
+
+/// simple stdin example from documentation but with python list append query/operation
```

Comment:
I don't follow the point of this test and how it differs from the [other one](https://github.com/getgrit/gritql/pull/573/files#diff-b90d6ee7389d6dd178a17e59f5fd798bb3799394b5c4367675321463f0ef7e10R2855).